code/_globalvars/global_lists.dm

```diff
@@ -15,6 +15,20 @@ GLOBAL_LIST_EMPTY(larva_burst_by_hive)
 GLOBAL_LIST_INIT_TYPED(custom_huds_list, /datum/custom_hud, setup_all_huds())
 GLOBAL_LIST_INIT_TYPED(custom_human_huds, /datum/custom_hud, setup_human_huds())
 
+//Basically counts how many times the Almayer was hit by each weapon. Loggin' wack.
+GLOBAL_LIST_INIT(ship_hits_tally, list(
+"times_hit_missile" = 0,
+"times_hit_railgun" = 0,
+"times_hit_odc" = 0,
+"times_hit_aaboiler" = 0,
+))
+
+//Ship's HP! ShipvsShip baybeee
+GLOBAL_LIST_INIT(ship_health_vars, list(
+"ship_hull_health" = 100,
+"ship_systems_health" = 100,
+))
```

Comment:
i feel like the ship being a datum is better

Reply:
An example of this specific code being datumized was here (but unfinished) https://github.com/cmss13-devs/cmss13/pull/3355 / https://github.com/cmss13-devs/cmss13/pull/3647